pkgs/os-specific/linux/radeontop/default.nix

```diff
@@ -3,11 +3,11 @@
 
 stdenv.mkDerivation rec {
   pname = "radeontop";
-  version = "2019-06-03";
+  version = "1.3";
```

Comment:
I'm not expert on how Nix is handling the sorting. At first glance it looks like that users will likely still end-up with `20xx` as it comes after `1.x`.

If I'm wrong then I will take it take and blame the Fedora package maintainer in me.

Reply:
That's incorrect, from Nix's POV packages with different versions are just different packages.

Reply:
To clarify, only one version of a package exists at a time in nixpkgs. If two versions need to exist, they're made into different packages and used accordingly (like `gcc`, `gcc7`, `gcc8`, etc)

Reply:
Thanks for the clarification.

Reply:
The sorting is important for people who use `nix-env -u`, that's the reason why I kept the date format even after radeontop started with named versions.